src/biokbase/narrative/jobs/job.py

```diff
@@ -10,6 +10,7 @@
 import biokbase.narrative.clients as clients
 from biokbase.narrative.app_util import map_inputs_from_job, map_outputs_from_state
 from biokbase.narrative.exception_util import transform_job_exception
+from biokbase.narrative.jobs.util import merge, merge_inplace
```

Comment:
Can you add `updated` to the `JOB_INIT_EXCLUDED_JOB_STATE_FIELDS` below --  it's not accurate or useful.

Reply:
Looks like this won't be a quick fix as it's at least somewhat baked into the tests

Reply:
Does that mean you aren't going to do it or just that I should wait to see it turn up in the PR?

Reply:
Yeah ... I don't think there's enough time